src/test/compile-fail/deriving-meta-unknown-trait.rs

```diff
@@ -11,7 +11,7 @@
 // ignore-tidy-linelength
 
 #[derive(Eqr)]
-//~^ ERROR cannot find derive macro `Eqr` in this scope
+//~^ ERROR cannot find macro `Eqr` in this scope
```

Comment:
I actually agree with this, and I was meaning to change it back. Should attribute macros show something different to "macro"?